wiki/Tournaments/IGTS/2018/en.md

```diff
@@ -0,0 +1,141 @@
+- [Challonge Brackets](https://challonge.com/u85xl34m)
+
+-----------------------
+
+## Participants
+
+*The participants will be listed once the registration phase is finished.*
+
+-----------------------
+
+## Mappool
+
+*The mappools will be revealed one week in advance for each stage.*
+
+------------------------------------------------------------------------
+
+## Ruleset
+### General Rules
+
+1. The Intermediate Global Taiko Showdown 2018 is a two-on-two Team Vs tournament, with the aspect of double-elimination bracket.
+
+2. In order for your registration to count, you must meet following conditions:
+   - Participant must fill out the provided form.
+   - Participant must be within rank **3,500 - 10,000 during registrations.**
+   - Participant are not allowed to rank up further than rank **2,500 during the tournament's occurrence.**
+   - Participant must join tournament's Discord server.
+3. The mapset will be announced the week before a stage begins.
+4. Match schedule is decided by the organizers, which will announce schedule the same day as the mapset every week. Each stage of this tournament will last a week, and that week starts when the mapset is announced a day before, so players need to get their match in before the week is over.
+5. If both team gets a tie on the TieBreaker, the TieBreaker will be replayed with FreeMod applied.
+6. If a player disconnects from the game, the opponent wins the round. 
+   - If the player disconnects during the first five seconds of the song, the match will be restarted.
+7. Maps cannot be used twice in the same round.
+8. If a player doesn't show up within **15 minutes** of the start time, the enemy wins by default.
+9.  Any other rules changes or unexpected occurrences will be announced in the tournament's forum thread.
+
+### Stage Regulations
+
+1. There are four stages to this tournament: Group Stage, Quarterfinals, Semifinals, and Finals.
+
+2. Matches are seeded. Teams will be seeded by performance points.
+3. In Group Stage, you need to win 4 beatmaps to win a match. (Best-of-7)
+4. In Round of 16, Quarterfinals, and Semifinals, you need to win 5 beatmaps to win a match. (Best-of-9)
+5. In Finals, you need to win 6 beatmaps to win a match. (Best-of-11)
+
+### Match Regulations
+1. A private lobby will be created for the match, and both teams will be invited into their match following the agreed schedule. Match will be played as **TeamVs and ScoreV2.**
+
+2. Each team is allowed to select one warm-up map.
+   - During Group Stage warm-ups must not exceed 4:00 minutes. In the following stages the warm-up can be up to 6 minutes long if both teams agree.
+   - Offensive songs may be blocked by referees.
+3. Map selection will be done by the team who is currently losing. At the start however, it will be whoever `!roll` the highest. When the starting team is decided, both teams will select one map to ban that is not the tiebreaker.
```

Comment:
`tiebreaker` -> `TieBreaker`

Note the capitalisation on the other occurrences.